tests/e2e/cron-jobs.spec.ts

```diff
@@ -6,6 +6,8 @@ import { AppModule } from '../src/app.module';
 import { CronService } from '../src/cron.service';
 import { nullPrototypeObjectProvider } from '../src/null-prototype-object.provider';
 
+const REMOVE_ANSI_CHARACTERS = /[\u001b\u009b][[()#;?]*(?:[0-9]{1,4}(?:;[0-9]{0,4})*)?[0-9A-ORZcf-nqry=><]/g
```

Comment:
I like regex but this one looks too cryptic :thinking: 

Also, I'd rename the variable because the regex itself doesn't removes anything. Maybe something like `RE_MATCH_ALL_ANS_CHARACTERS`